webapp/src/ts/components/actionbar/actionbar.component.html

```diff
@@ -61,58 +66,68 @@
+              </li>
+            </ul>
 
-          <a *ngIf="actionBar?.left?.childPlaces?.length === 1"
-             mmAuth="can_create_places"
-             class="mm-icon mm-icon-inverse mm-icon-caption"
-             [routerLink]="actionBar.left.userFacilityId ? ['/', 'contacts', actionBar.left.userFacilityId, 'add', actionBar.left.childPlaces[0]?.id] : ['/', 'contacts', 'add', actionBar.left.childPlaces[0]?.id]"
-             [queryParams]="{ from: 'list' }"
-          >
-            <span class="fa-stack">
-              <i [innerHTML]="actionBar.left.childPlaces[0].icon | resourceIcon"></i>
-              <i class="fa fa-plus fa-stack-1x"></i>
-            </span>
-            <p>{{ actionBar.left.childPlaces[0].create_key | translate}}</p>
-          </a>
+            <a
+              *ngIf="actionBar?.left?.childPlaces?.length === 1"
+              mmAuth="can_edit,can_create_places"
+              class="mm-icon mm-icon-inverse mm-icon-caption create-place"
+              [routerLink]="actionBar.left.userFacilityId ? ['/', 'contacts', actionBar.left.userFacilityId, 'add', actionBar.left.childPlaces[0]?.id] : ['/', 'contacts', 'add', actionBar.left.childPlaces[0]?.id]"
+              [queryParams]="{ from: 'list' }"
+            >
+              <span class="fa-stack">
+                <i [innerHTML]="actionBar.left.childPlaces[0].icon | resourceIcon"></i>
+                <i class="fa fa-plus fa-stack-1x"></i>
+              </span>
+              <p>{{ actionBar.left.childPlaces[0].create_key | translate}}</p>
+            </a>
 
-          <a *ngIf="actionBar?.left?.exportFn"
-             mmAuth [mmAuthOnline]="true" [mmAuthAny]="['can_export_all', 'can_export_contacts']"
-             class="mm-icon mm-icon-inverse mm-icon-caption"
-             (click)="actionBar?.left?.exportFn()"
-             [ngClass]="{ 'mm-icon-disabled': !actionBar.left.hasResults }"
-          >
-            <span class="fa fa-arrow-down"></span>
-            <p>{{'Export' | translate}}</p>
-          </a>
+            <a *ngIf="actionBar?.left?.exportFn"
+               mmAuth [mmAuthOnline]="true" [mmAuthAny]="['can_export_all', 'can_export_contacts']"
+               class="mm-icon mm-icon-inverse mm-icon-caption"
+               (click)="actionBar?.left?.exportFn()"
+               [ngClass]="{ 'mm-icon-disabled': !actionBar.left.hasResults }"
+            >
+              <span class="fa fa-arrow-down"></span>
+              <p>{{'Export' | translate}}</p>
+            </a>
+          </div>
         </div>
       </div>
 
       <div class="col-sm-4 general-actions left-pane" *ngIf="currentTab === 'messages'">
-        <div class="actions dropup">
-          <a class="mm-icon mm-icon-inverse mm-icon-caption send-message" *ngIf="actionBar?.left?.openSendMessageModal" (click)="actionBar?.left?.openSendMessageModal($event)">
+        <div class="actions dropup" mmAuth [mmAuthAny]="['can_edit', 'can_export_all', 'can_export_messages']">
```

Comment:
Same here.

Reply:
Thanks. Export might not be dependent on `can_edit`